### DISCUSSION TITLE
Domoticz

### DISCUSSION THREAD

**Discussion by RabbitMD** [fsaris/home-assistant-zonneplan-one#148] [Q&A]
Hi, i am rewriting this module so that it can be used in domoticz aswell.
authentication works etc , but calling url = f"{base_url}/connection/{uuid}/pv-installation" results in an 404 error (message not found)
any idea how to solve this?

**Comment by fsaris** [comment]
> Hi, i am rewriting this module so that it can be used in domoticz aswell

Nice

> `f"{base_url}/connection/{uuid}/pv-installation`

I think you mis a 's' in `connections`

**Reply by RabbitMD** [reply]
Thanks, i tried different urls and paths also with connections but somehow it keeps giving that 403 aka message not found :(

**Reply by fsaris** [reply]
404 is invalid url/path
403 is no access (so invalid token etc, maybe the response message tells you more).

I found it easiest to test the calls with postman

**Reply by RabbitMD** [reply]
yeah i am using postman also, are you willing to share the endpoints / url's?

**Reply by RabbitMD** [reply]
i think if figured it out, i am using the wrong uuid ;)

**Reply by VincentvdH-91** [reply]
Hi @RabbitMD would you be able to share how you got around this? I'm trying to get this working too, but run into similar issues.

**Comment by RabbitMD** [comment]
@VincentvdH-91 I used the wrong uuid, when accessing the endpoint /user-accounts/me you get several UUID in the response : 1 for your account (which i was using) one just in front of [connections] (should be the same as the one in [connections] depending if you have multiple setups. When i used the uuid thats in [connections] it started to work.
